code/modules/vehicles/mecha/equipment/mecha_equipment.dm

```diff
@@ -21,7 +21,7 @@
 	///Bitflag. Determines the range of the equipment.
 	var/range = MECHA_MELEE
 	/// Bitflag. Used by exosuit fabricator to assign sub-categories based on which exosuits can equip this.
-	var/mech_flags = NONE
+	var/mech_flags = EXOSUIT_MODULE_ALL
```

Comment:
It's probably a good idea to just use the `ALL` define. That way people in the future don't have to keep adding their flags to the `EXOSUIT_MODULE_ALL` define.

Reply:
```suggestion
	var/mech_flags = ALL
```